docs/direct-debits/How-To-Request-Payments.md

```diff
@@ -24,7 +24,8 @@ These are most commonly caused by incorrect bank account details. In this case t
 
 ## Creating a Payment
 
-You can create a payment either as a single one-off or as a recurring subscription. You must have created a portfolio and mandate before you can create a payment or subscription. The mandate must also be `Active`. Multiple payments and subscriptions or a mixture of both can all be raised against a single mandate.
+You can create a payment either as a single one-off or as a recurring subscription. You must have created a portfolio and mandate before you can create a payment or subscription. The mandate must also be `Active`. Multiple payments and subscriptions or a mixture of both can all be raised against a single mandate. Refer to the Direct debit <a href="./Timings.md">timings</a> guide for more information about the timings related to payments.
```

Comment:
The link to Timings still appear to be broken. Maybe this format will fix it: 
```[timings](./Timings.md)``` instead of ```<a href...>timings<a>```